text/fuzz/corpus/from_test_parse_error_0

```diff
@@ -0,0 +1 @@
+bla 3.14
```

Comment:
Thanks a bunch for the effort!
Great work @beorn7!

On Thu, Jul 30, 2015, 10:46 PM Morten Siebuhr <notifications@github.com>
wrote:

> In text/fuzz/corpus/from_test_parse_error_0
> <https://github.com/prometheus/client_golang/pull/151#discussion_r35918355>
> :
>
> > @@ -0,0 +1 @@
> > +bla 3.14
>
> All the from_test_parse_ are written out as a one-time job from the
> test-suite, so it should be a clone of whatever was in there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/client_golang/pull/151/files#r35918355>.
>